src/main/java/dev/projectg/geyserhub/module/bossBar/BossBarListener.java

```diff
@@ -0,0 +1,66 @@
+package dev.projectg.geyserhub.module.bossBar;
+import org.bukkit.Bukkit;
+import org.bukkit.boss.BarColor;
+import org.bukkit.boss.BarStyle;
+import org.bukkit.boss.BossBar;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerJoinEvent;
+
+public class BossBarListener implements Listener {
+    @EventHandler
+    public void onJoin(PlayerJoinEvent e){
+        for(Integer index : BossBarHandler.color.keySet()) {
+            if(getColor(BossBarHandler.color.get(index)) == null) {
+                BossBar bossBar = Bukkit.createBossBar("§cThis color is not available!", BarColor.WHITE, BarStyle.SOLID);
+                bossBar.addPlayer(e.getPlayer());
+            }else {
+                if(getBarStyle(BossBarHandler.style.get(index)) == null) {
+                    BossBar bossBar = Bukkit.createBossBar("§cThis BarType is not available!", BarColor.WHITE, BarStyle.SOLID);
+                    bossBar.addPlayer(e.getPlayer());
+                    return;
```

Comment:
probably should make a warn log in the console if the config isn't right, and check validity on load

Reply:
Yea sounds good, for most parts like error on color or type we send an error that the format isn't good